src/components/node-list/node-list-tree.js

```diff
@@ -111,16 +115,25 @@ const TreeListProvider = ({
     }
   };
 
-  const renderModularPipelines = (treeData) =>
-    treeData.children.map((node) =>
+  const renderModularPipelines = (treeData, parentStatus) => {
```

Comment:
Is this status the focused status of a parent? Or the disabled status? What are the possible values?

Reply:
yes it is the focused Status of the parent pipeline, which is used to determine the display state of the pipeline - the possible values are only `true` or `false`

- if any higher parent is selected for focusMode, the parentStatus will remain true for all child modular pipelines down all levels. 
- If this level is true, then the label of that modular pipeline row will be rendered in a 'non-disabled' way while still having disabled status